.github/workflows/all-in-one-build.yml

```diff
@@ -42,7 +42,9 @@ jobs:
 
     - name: Export BRANCH variable for pull_request event
       run: |
-        echo "BRANCH=${GITHUB_HEAD_REF}" >> $GITHUB_ENV
+        export BRANCH=${GITHUB_HEAD_REF}
+        echo "we are on branch=$BRANCH"
+        echo "BRANCH=${BRANCH}" >> $GITHUB_ENV
```

Comment:
The next step could also echo the value

Reply:
sure, I will do that while moving this to its own action as you suggested. Since it can be shared with other docker build and push jobs for all jaeger components.